assets/docs/fragments/cli-installation.md

```diff
@@ -0,0 +1,62 @@
+### CLI Installation
+Follow the [AsyncAPI CLI installation](https://github.com/asyncapi/cli#installation) instructions below, based on your computer’s operating system.
+
+<details>
+<summary>MacOS</summary>
+
+`brew`
+</br>
+You can install in MacOS via brew: `brew install asyncapi`.
+
+`pkg`
+</br>
```

Comment:
```suggestion
<br/>
You can install in MacOS via brew: `brew install asyncapi`.

`pkg`
<br/>
```

otherwise docs generation fails when you try to use it. JSX-related issue